workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -463,19 +470,116 @@ pre[class*='language-'] {
 			border-bottom: solid;
 			border-color: var(--color-action-neutral-hover-10, #eaecee);
 		}
+
+		.learn-article-nav-item {
+			padding: 4.8px var(--spacer-3, 1rem);
```

Comment:
fraction pixels seem very odd, we should use whole numbers

Reply:
fixed on new PR https://github.com/liferay-customer-delivery/liferay-portal/pull/89